controllers/controllers.js

```diff
@@ -81,5 +83,28 @@ exports.getCommentsC = (req,res,next) => {
     .catch((err) => {
       next(err);
     });
-  
-}
\ No newline at end of file
+};
+
+exports.postCommentsC = (req, res, next) => {
+  const id = req.params.review_id;
+  const { username, body } = req.body;
+  getReviewsByIdM(id)
```

Comment:
Works well, but you could use a Promise.all to avoid chaining .then blocks to achieve this (which should be done in the model rather than the controller).
You also only need 1 catch block in the controller at the end of a promise chain to catch all errors.